openerp/openupgrade/doc/source/analyse.rst

```diff
@@ -1,29 +1,25 @@
-How to run your own analysis
-============================
-If you do need to run your own analysis, you need to perform the following
-steps (the awkward processing of the server log file is now obsolete).
+Run your own analysis
+=====================
 
-* Set up two OpenUpgrade servers of subsequent Odoo releases
+* First you need to prepare your python environment by installing the appropiate release of [openupgradelib](https://github.com/OCA/openupgradelib/releases)
 
-* On both instances, install a database without demo data and
-  install the *openupgrade_records* module, which is included in the
-  OpenUpgrade server distribution. This will add a menu
-  *OpenUpgrade Development* to the Administration menu.
+* In order to analyze the difference of two codebases you need to specially prepare your different codebases by applying this OpenUpgrade patch onto your odoo code-base. (@ProjectLeader: Please provide pathces towards OCA/OCB and standard Odoo for different versions) This will add some folders of the OpenUpgrade tool suite and modify some odoo core files in order to work smoothly with OpenUpgrade.
```

Comment:
It's good to make the explanation clearer, but removing the bullets makes it less readable